bigbluebutton-html5/imports/api/annotations/addAnnotation.js

```diff
@@ -1,7 +1,6 @@
 import { check } from 'meteor/check';
-import _ from 'lodash';
```

Comment:
removing lodash import causes an error `Uncaught (in promise) ReferenceError: _ is not defined on /imports/api/annotations/addAnnotation.js:21`

![Screenshot from 2023-05-05 09-30-45](https://user-images.githubusercontent.com/3728706/236461076-6cdabc31-b73e-4edf-aacc-834d6f13e0df.png)

Reply:
Resolved